src/build/webpack.daemon.config.js

```diff
@@ -19,33 +22,24 @@ module.exports = {
   },
   devtool: "source-map",
```

Comment:
@DanazSdiq please could you add the following to the config. We don't need to minimise, and this will make debug easier

```js
  optimization: {
    minimize: false,
  },
```